Formula/osmfilter.rb

```diff
@@ -17,10 +17,12 @@ class Osmfilter < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "zlib" => :build
+  uses_from_macos "zlib"
```

Comment:
```suggestion
  uses_from_macos "zlib" => :build
```

Reply:
It looks like it links with zlib too so I'd drop the `=> :build` part:

```
$ brew linkage osmfilter
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libz.1.dylib
```